### PR TITLE
Features/vsts new sdk port

### PR DESCRIPTION
**Pull request by harshil93**
Porting the the task to use the new sdk and adding some tests

### CODE REVIEW

**Comment by pascalberger** [comment]
@harshil93 Shouldn't the old PS handler be removed? Or do you plan to maintain two versions. Sounds like too much trouble to me to keep them in sync.

**Comment by harshil93** [comment]
I will remove it after a month or so if no issues came up. People using TFS update 2 might not have the latest handler.

via CloudMagic Email<https://cloudmagic.com/k/d/mailapp?ct=pa&cv=8.5.49&pv=6.0.1&source=email_footer_2>
On Sun, Jul 24, 2016 at 11:11 PM, Pascal Berger <notifications@github.com<mailto:notifications@github.com>> wrote:


@harshil93<https://github.com/harshil93> Shouldn't the old PS handler be removed? Or do you plan to maintain two versions. Sounds like too much trouble to me to keep them in sync.

-

**Comment by pascalberger** [comment]
@harshil93 Any plans to merge this soon, as I'm waiting for this to fix #24?

File: Utilites/Tokenizer/tokenize-ps3.ps1 [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+param()
+
+Trace-VstsEnteringInvocation $MyInvocation
+
+try {
+    [string]$SourcePath = Get-VstsInput -Name SourcePath
+    [string]$DestinationPath = Get-VstsInput -Name DestinationPath
+    [string]$ConfigurationJsonFile = Get-VstsInput -Name ConfigurationJsonFile
+    . $PSScriptRoot\Helpers.ps1
+    Write-Verbose "Entering script tokenize.ps1"
```
Comment (pascalberger) [inline Utilites/Tokenizer/tokenize-ps3.ps1]: This seems to be redundant to the Trace-VstsEnterinigInvocation call above

**Pull request merged**